ts/test-both/util/getInitials_test.ts

```diff
@@ -0,0 +1,53 @@
+// Copyright 2021 Signal Messenger, LLC
+// SPDX-License-Identifier: AGPL-3.0-only
+
+import { assert } from 'chai';
+
+import { getInitials } from '../../util/getInitials';
+
+describe('getInitials', () => {
+  it('returns undefined when passed undefined', () => {
+    assert.isUndefined(getInitials(undefined));
+  });
+
+  it('returns undefined when passed an empty string', () => {
+    assert.isUndefined(getInitials(''));
+  });
+
+  it('returns undefined when passed a string with no letters', () => {
+    assert.isUndefined(getInitials('123 !@#$%'));
+  });
+
+  it('returns the first letter of a name that is one ASCII word', () => {
+    assert.strictEqual(getInitials('Foo'), 'F');
+  });
+
+  [
+    'Foo Bar',
+    'foo bar',
+    'F Bar',
+    'Foo B',
+    'FB',
+    'F.B.',
+    '0Foo 1Bar',
+    "Foo B'Ar",
+    'Foo Q Bar',
+    'Foo Q. Bar',
+    'Foo Qux Bar',
+    'Foo "Qux" Bar',
+    'Foo-Qux Bar',
+    'Foo Bar-Qux',
+    'Foo Bar Jr.',
+    'Foo Bar, Sr.',
+    'Foo Bar II',
+    'Foo Bar, IV',
+  ].forEach(name => {
+    it(`returns 'FB' for '${name}'`, () => {
+      assert.strictEqual(getInitials(name), 'FB');
+    });
+  });
+
+  it('returns initials with diacritical marks', () => {
+    assert.strictEqual(getInitials('Ḟoo Ḅar'), 'ḞḄ');
+  });
+});
```

Comment:
Could you add some tests for names in other languages? I think testing a Korean name like `공효진` and a French name `Jacques d'Allonville` would be useful. It could also be good to figure out what should happen for right-to-left languages like Arabic.

Reply:
I added tests for letters from non-Latin alphabets and right-to-left languages. Also added an exemplary name with an apostrophe in the last name.